src/Core/Domain/ProjectModel.ts

```diff
@@ -28,3 +28,45 @@ export interface FlatEnvironment {
   description?: string;
   icon?: string;
 }
+export interface EnvironmentExpertOnly {
```

Comment:
Same remark as in the other PR, since both interfaces have the same base, maybe extending is a good idea